template/src/Autovis/Controls.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import "./controls.css";
+import pad from './pad';
+
+
+const printTime = (currentTime) => {
```

Comment:
Let's make this a reusable export in another file

Reply:
the whole thing? Sure can do boss

Reply:
should it just have the pad function in it, or should pad be in another file as well?

Reply:
pad should be in another file